src/bb2.py

```diff
@@ -2,13 +2,90 @@
 Blue Button 2.0 SDK Class
 
 """
+import json
+import os
+import pathlib
+import yaml
+
+ROOT_DIR = os.path.abspath(os.curdir) + "/"
+DEFAULT_CONFIG_FILE_LOCATION = ROOT_DIR + "./.bluebutton-config.json"
+
+ENVIRONMENT_URLS = {
+    "SANDBOX": "https://sandbox.bluebutton.cms.gov",
+    "PRODUCTION": "https://api.bluebutton.cms.gov",
+}
 
 
 class Bb2:
     name = "bb2"
     verbose_name = "Blue Button 2.0 SDK Package"
-    
-    def hello(self):
-        mesg = "Hello from BB2 SDK Class method!!!"
-        print(mesg)
-        return mesg
\ No newline at end of file
+
+    def __init__(self, config_file=DEFAULT_CONFIG_FILE_LOCATION):
+        self.config_file = config_file
+        self.config = {}
+
+        self.environment = None
```

Comment:
Do we actually need to keep track of the env? I know for the node SDK I just used it to generate the base url but never actually saved it.

Reply:
@nbragdon Good idea to not save this, since it is only used for the base url. I'll remove this.  TY!

Reply:
@nbragdon This is removed.